Makefile

```diff
@@ -3,9 +3,10 @@ PKG := github.com/btcsuite/btcwallet
 LINT_PKG := github.com/golangci/golangci-lint/cmd/golangci-lint
 GOACC_PKG := github.com/ory/go-acc
 GOIMPORTS_PKG := golang.org/x/tools/cmd/goimports
+TOOLS_DIR := tools
 
+GOPATH := $(shell go env GOPATH)
```

Comment:
nit: `LINT_COMMIT` below is now unused.
It would also be nice if we could remove the `GOACC_COMMIT` and `GOIMPORTS_COMMIT` values as well by creating a `go.mod` file in the `tools` directory that pins these versions, then just install them from the `tools` directory as we do here: https://github.com/lightninglabs/taproot-assets/blob/main/Makefile#L79